front/components/assistant/conversation/ConversationMessage.tsx

```diff
@@ -0,0 +1,24 @@
+import { Avatar } from "@dust-tt/sparkle";
+
+/**
+ * Parent component for both UserMessage and AgentMessage, to ensure avatar,
+ * side buttons and spacing are consistent between the two
+ */
```

Comment:
Why not have the name of the author as well?

Reply:
Note: The avatar component needs image URL but also name if it's not set so that it can render initials

Reply:
Also, are we sure we want to have the avatar be a react node and not juste a name + image profile? Are we going to inject the avatar component? Looks like  this could create some rendering issue if the right component is not injected + it makes it harder to use.

LMK if I missed something 🙏

Reply:
@philipperolet are you planning on keeping the API as is for avatar or are we switching to just the profile URL and call the avatar component directly in there?

Reply:
just profileUrl + name